rotkehlchen/api/v1/resources.py

```diff
@@ -3378,3 +3380,26 @@ def post(self, from_timestamp: Timestamp, to_timestamp: Timestamp) -> Response:
             from_ts=from_timestamp,
             to_ts=to_timestamp,
         )
+
+
+class TimestampHistoricalBalanceResource(BaseMethodView):
+
+    query_schema = HistoricalBalanceSchema()
+    post_schema = HistoricalPerAssetBalanceSchema()
+
+    @require_premium_user(active_check=False)
+    @use_kwargs(query_schema, location='json')
```

Comment:
Though with the new request this would go away

```suggestion
    @use_kwargs(put_schema, location='json')
```